recipes/perl4-corelibs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "perl4-corelibs" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: http://search.cpan.org/CPAN/authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.003.tar.gz
+  md5: f74747c234470a087871f564a5f5db15
+
+build:
+  number: 0
+  script: "cpanm ."
+
+requirements:
+  build:
+    - perl-threaded
+    - perl-app-cpanminus
+    - perl-module-build
+
+  run:
+    - perl-threaded
+
+
+test:
+  commands:
+    - perl -e 'require "getopts.pl"' 
+
+about:
+  home: http://search.cpan.org/~zefram/Perl4-CoreLibs-0.003/lib/Perl4/CoreLibs.pm
+  description: "This is a collection of .pl files that have historically been bundled with the Perl 
+                core but are planned not to be so distributed with core version 5.15 or later." 
+
+
```

Comment:
Can you remove the trailing line breaks? Also please include a license information.